### ISSUE TITLE
Android streaming

### ISSUE THREAD

**Issue by Zwem** [buhe/react-native-piliv2#1]
When i try run streaming on android, throw error "You must initialize SrteamingEnv by StreamingEnv#init first!"

**Comment by buhe** [comment]
Sorry for later, add 
```
@Override
    public void onCreate() {
        super.onCreate();
        StreamingEnv.init(this.getApplicationContext());
    }
```

to MainApplication.java

I will add that code to react-native-pili on next version.

**Comment by buhe** [comment]
I develop a [starter-kit](https://github.com/buhe/pili-startkit) for this , use cocoapods to help integration iOS pili-core-sdk easily.

**Comment by Zwem** [comment]
Thank you for answer, What about this issue https://github.com/buhe/react-native-pili/issues/13, i have same when use react-native-piliv2

**Comment by buhe** [comment]
I fixed at 2.1.2 ,please upgrade .
